yarn.lock

```diff
@@ -3013,10 +3013,10 @@ box-node-sdk@^1.33.0:
     url-template "^2.0.8"
     uuid "^3.3.3"
 
-box-ui-elements@^12.1.0-beta.42:
-  version "12.1.0-beta.42"
-  resolved "https://registry.yarnpkg.com/box-ui-elements/-/box-ui-elements-12.1.0-beta.42.tgz#0da6793ad769b5cc348acb9ad57d7fbf155b7969"
-  integrity sha512-MfmUhSD2E0fog0017iQiP2IhivketGrjvesyHPCD4F+rx7H59PWo8y3U4BbPaxWNHHsDqNXonynwU2HreUR3FA==
+box-ui-elements@^12.1.0-beta.43:
```

Comment:
Is this upgrade needed as part of this change? Should it be done as a separate PR?